packages/atomic/src/components/atomic-result/atomic-result-cell-mobile.pcss

```diff
@@ -0,0 +1,192 @@
+
+    atomic-result-section-title-metadata {
+      font-size: 0.75rem;
+      line-height: 1rem;
+    }
+
+    atomic-result-section-emphasized {
+      font-weight: 500;
+      font-size: 1.5rem;
+      line-height: 2rem;
+    }
+
+    atomic-result-section-excerpt {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      margin-top: 1.25rem;
+    }
+
+    atomic-result-section-bottom-metadata {
+      font-size: 0.75rem;
+      line-height: 1rem;
+      margin-top: 1.25rem;
+    }
+  }
+
+  &.density-normal {
+    atomic-result-section-visual {
+      margin-bottom: 0.75rem;
+    }
+
+    atomic-result-section-badges {
+      font-size: 0.75rem;
+      line-height: 1rem;
+      margin-bottom: 1rem;
+      height: 2rem;
+    }
+
+    atomic-result-section-actions {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      margin-top: 1rem;
+      height: 2rem;
+    }
+
+    atomic-result-section-title {
+      font-size: 1rem;
+      line-height: 1.5rem;
+      height: 3rem;
+    }
+
+    atomic-result-section-title-metadata {
+      font-size: 0.75rem;
+      line-height: 1rem;
+    }
+
+    atomic-result-section-emphasized {
+      font-weight: 500;
+      font-size: 1.5rem;
+      line-height: 2rem;
+    }
+
+    atomic-result-section-excerpt {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      margin-top: 1rem;
+    }
+
+    atomic-result-section-bottom-metadata {
+      font-size: 0.75rem;
+      line-height: 1rem;
+      margin-top: 1rem;
+    }
+  }
+
+  &.density-compact {
+    atomic-result-section-visual {
+      margin-bottom: 0.5rem;
+    }
+
+    atomic-result-section-badges {
+      font-size: 0.75rem;
+      line-height: 1rem;
+      margin-bottom: 1rem;
+      height: 2rem;
+    }
+
+    atomic-result-section-actions {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      margin-top: 0.75rem;
+      height: 2rem;
+    }
+
+    atomic-result-section-title {
+      font-size: 1rem;
+      line-height: 1.5rem;
+      height: 3rem;
+    }
+
+    atomic-result-section-title-metadata {
+      font-size: 0.75rem;
+      line-height: 1rem;
+    }
+
+    atomic-result-section-emphasized {
+      font-weight: 500;
+      font-size: 1.5rem;
+      line-height: 2rem;
+    }
+
+    atomic-result-section-excerpt {
+      font-size: 0.875rem;
+      line-height: 1.25rem;
+      margin-top: 0.75rem;
+    }
+
+    atomic-result-section-bottom-metadata {
+      font-size: 0.75rem;
+      line-height: 1rem;
+      margin-top: 0.75rem;
+    }
+  }
+
+  /* == Image styles == */
+  &.image-small {
+    atomic-result-section-visual {
+      width: 9.5rem;
+      height: 9.5rem;
+    }
+  }
+
+  &.image-icon {
+    atomic-result-section-visual {
+      width: 2rem;
+      height: 2rem;
+    }
+  }
+
+  &.image-none {
+    atomic-result-section-visual {
+      display: none;
+    }
+  }
+
+  /* == Common styles == */
+  atomic-result-section-badges,
+  atomic-result-section-actions {
+    overflow-x: auto;
+  }
```

Comment:
I diverge from the design here. On mobile, I made badges and actions scrollable horizontally.

In pretty much every other layout, the badge and action sections have the height of exactly one button, which means setting `height: 100%` on each button is the easiest way to have buttons that scale with different layouts.

However, I didn't like having mobile grids be an edge-case, and enforcing a 0.5rem padding vertically between buttons while also supporting a variable amount of buttons with variable widths would be overly complicated. If we need to, we should be able to change this later.

Reply:
I'd prefer to have them one on top of the other, but I understand it's a bit of a corner case, we can see how this evolve and get feedback from Johnny